website/docs/r/independent_disk.html.markdown

```diff
@@ -48,7 +47,46 @@ The following arguments are supported:
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
 * `name` - (Required) Disk name
-* `size` - (Required) - Size of disk in MB
-* `bus_type` - (Optional) - Disk bus type. Values can be: IDE, SCSI, SATA 
-* `bus_sub_type` - (Optional) - Disk bus subtype. Values can be: "IDE" for IDE. buslogic, lsilogic, lsilogicsas, VirtualSCSI for SCSI and ahci for SATA
-* `storage_profile` - (Optional) - The name of storage profile where disk will be created
\ No newline at end of file
+* `size` - (Optional; **Deprecated**) Size of disk in MB
```

Comment:
A note about "read" no updating this value?

Reply:
added  additional note.